wandb/integration/keras/callbacks/model_logger.py

```diff
@@ -0,0 +1,27 @@
+import wandb
+
+import tensorflow as tf
+from tensorflow.keras import callbacks
+
+from ..keras import patch_tf_keras
+
+patch_tf_keras()
+
+
+class ModelLogger(callbacks.Callback):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._graph_rendered = False
+
+    def _render_graph(self):
+        if not self._graph_rendered:
+            wandb.run.summary["graph"] = wandb.Graph.from_keras(self.model)
```

Comment:
I think we should have a `try-except` clause here. Check out this jira [ticket ](https://wandb.atlassian.net/browse/WB-10266?focusedCommentId=46319&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-46319) for the edge case.